cmd/spec/path.yaml

```diff
@@ -289,3 +289,23 @@ paths:
                 $ref: '#/components/schemas/v1.NoopReservationResponse'
         "500":
           $ref: '#/components/responses/InternalError'
+  /availability_status/sources:
+    post:
+      operationId: availabilityStatus
+      description: >
+        Schedules a background operation of Sources availability check. These checks are
+        are performed in separate process at it's own pace. Results are sent via Kafka
+        to Sources. There is no output from this REST operation available, no tracking
+        of jobs is possible.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/v1.AvailabilityStatusRequest'
+        description: availability status request with source id
+        required: true
+      responses:
+        '200':
```

Comment:
I think @akhil-jha prefers it to be 204 because of the empty response

Reply:
@lzap @adiabramovitch did we adress this, or it was not issue in IQE this time @akhil-jha ?